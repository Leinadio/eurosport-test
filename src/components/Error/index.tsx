import React from 'react';
import './styles.css'

export default function Error(): JSX.Element {
  return (
    <p className="error-text">An error has appeared</p>
  )
}
