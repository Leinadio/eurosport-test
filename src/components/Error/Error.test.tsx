import React from 'react';
import renderer from 'react-test-renderer';
import Error from './'

describe('Error', function () {
  it('should display Error', () => {
    const tree = renderer
      .create(<Error />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});