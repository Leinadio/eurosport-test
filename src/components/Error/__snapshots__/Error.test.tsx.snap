// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should display Error 1`] = `
<p
  className="error-text"
>
  An error has appeared
</p>
`;
